### ISSUE TITLE
[BUG] Input field for repository URL is code page dependend

### ISSUE THREAD

**Issue by mgrossmann** [wizardofzos/zigi#390]
**Describe the bug**
When I set up a repository using a non-English code page, the @ character in the URL (or any other special char) seems to be misinterpreted. 

**Release of zigi**
V3R2

**To Reproduce**
1.  set terminal emulator to  1141
2.  try to clone a repo

**Screenshot** 
![image](https://user-images.githubusercontent.com/784509/105004706-a05a3e00-5a34-11eb-9ee7-88525aacaea6.png)

Do I change the code page back to 1146, the clone is working fine. But, in my case,  I then ran into issue #389 

regards 
Mike

**Comment by lbdyck** [comment]
I'm going to have to defer to Henri on this one.

**Comment by cietpfm** [comment]
z/OS UNIX by default uses the EBCDIC codepage IBM-1047

Code page [1141](https://www.ibm.com/support/knowledgecenter/SSEQ5Y_14.0.0/com.ibm.pcomm.doc/reference/html/hcp_reference35.htm) encodes the '@' character differently than code page [1047](https://www.ibm.com/support/knowledgecenter/SSEQ5Y_5.9.0/com.ibm.pcomm.doc/reference/html/hcp_reference27.htm)

It is my understanding that Zigi pannels do not use [extended code page support](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.4.0/com.ibm.zos.v2r4.f54dg00/cecps.htm
)

**Comment by mgrossmann** [comment]
@cietpfm yes this is the case. But this makes it nearly unusable outside the us.
